facade-app/src/test/Utils.test.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  // getResource,
+  removeChildByParentElementId,
+  appendStaticResourcesToDocument,
+  manipulateDOM,
+} from "../Utilities/Utils";
+import { ResourceType } from "../interface/State";
+
+// interface CallbackParams {
```

Comment:
I think we can remove these commented tests.

Reply:
I will remove it.